src/gd_tiff.c

```diff
@@ -957,6 +961,13 @@ BGD_DECLARE(gdImagePtr) gdImageCreateFromTiffCtx(gdIOCtx *infile)
 		goto error;
 	}
 
+	if (TIFFGetField (tif, TIFFTAG_XRESOLUTION, &res_float)) { 
```

Comment:
no space before the "(" after "TIFFGetField".  please match style of the file you're editing.